package.json

```diff
@@ -57,7 +61,21 @@
         "node": ">=4.2.1"
     },
     "scripts": {
-        "test": "node tests/all-tests.js",
+        "pretest": "npm run pretest:clean && npm run pretest:instrument && npm run pretest:copyUninstrumentedCode",
+        "pretest:clean": "node node_modules/rimraf/bin.js reports/* coverage/* instrumented/*",
+        "pretest:instrument": "npm run pretest:instrumentNodeModules && npm run pretest:instrumentConfigs",
+        "pretest:instrumentNodeModules": "node node_modules/istanbul/lib/cli.js instrument --complete-copy -o instrumented/universal/gpii/node_modules gpii/node_modules -x '**/lib/**' -x 'testing/**' -x 'tests/**'",
+        "pretest:instrumentConfigs": "node node_modules/istanbul/lib/cli.js instrument --complete-copy -o instrumented/universal/gpii/configs gpii/configs",
+        "pretest:copyUninstrumentedCode": "node scripts/copyUninstrumentedCode.js",
+        "test": "npm run test:node && npm run test:browser",
+        "test:browser": "npm run pretest:instrument && npm run pretest:copyUninstrumentedCode && node node_modules/testem/testem.js ci --file tests/testem.js --skip PhantomJS",
```

Comment:
Hi, @cindyli.  When running the node and browser tests, no coverage report is generated.  `npm test` also runs the `posttest` scripts, which generate the coverage report.

If the user manually runs the relevant `posttest` script, they may get an incomplete picture, or may have multiple runs that exercise the same code (in effect multiplying the coverage counts).  The docs only indicate two ways to run the coverage report, both of which are not affected by these issues.  I hope that's sufficient.